jbehave-support-core/docs/Expression-commands.md

```diff
@@ -185,16 +185,27 @@ This command simply do the sum of parameters.
 > Result of the command: `7`
 
 #### RANDOM_DATE
-Generates random date in range 1970 - 2059.
+Generates random date in range 1970 - 2059 by default.
+Takes up to 2 optional parameters to specify start and/or end date in several formats:
+- YYYY
+- YYYY.M
+- YYYY.M.D
 
 > Example:
 > ```
 > {RANDOM_DATE}
+> {RANDOM_DATE:1999}        //Random date since 1.1.1999 till 31.12.2059
+> {RANDOM_DATE:2000.5}      //Random date since 1.5.2000 till 31.12.2059
+> {RANDOM_DATE:2001.6.7}    //Random date since 7.6.2001 till 31.12.2059
+> {RANDOM_DATE:2002:2003}   //Random date since 1.1.2002 till 1.1.2003
+> {RANDOM_DATE:2002.4.5:2003.8.9} //Random date since 5.4.2002 till 9.8.2003
 > ```
 > Result of the command: `LocalDate object`
 
 #### RANDOM_DATE_TIME
-Generates random datetime in range 1970 - 2059.
+Generates random datetime in range 1970 - 2059 be default.
+Takes up to 2 optional parameters to specify start and/or end date see [RANDOM_DATE](#random_date).
+Time range is not implemented yet, please rise request if you see 
```

Comment:
`If you see`..... if you see what?

Reply:
if you see :smile:  If you are blind, don't rise request :rofl: TODO.